### ISSUE TITLE
Posting as accessibility because now it really is so - installation process password validation is very bad!

### ISSUE THREAD

**Issue by Milo-W** [joomla/joomla-cms#47730]
### Steps to reproduce the issue

1. Try to install Joomla 6.1.0.
2. Admin user step - password is yellow, goes through.
3. Database creation step - you go through.
4. Now it shoves an error message that the user password wasn't hard enough. Just like that, one step further from the actual place it happened!

### Version

6.1

### Expected result

At the admin account password input screen it should validate properly and if the password is too weak, it shouldn't allow to go further.

BUT

even if it goes through, there is still no "back" button on the validation screen (step 4 in my workflow listing above).
- this screen shouldn't be here anyway, it should all be done on the password input screen and it should hold the user THERE.

### Actual result

You get a warning screen, no buttons, nothing to do - and the previous / next buttons in the browser STILL go to previous / next pages, not the process! Ideally, it should go to the previous step! (of course, it should be the admin password input screen - or best: there shouldn't be a separate error page!)

Also, this is a copy of https://github.com/joomla/joomla-cms/issues/46111
which either wasn't understood by the kind developers or something else went wrong.

### What device were you using? (eg, phone, laptop)

I think it's device independent.
A desktop monitor though.

### What web browser were you using? (eg, Chrome, Safari)

Edge, Windows 10

### Assistive Technology?

nope, pure heuristics/WCAG problem here

### Additional Comments

This is outrageous as it's a bug that wasn't really debugged! see https://github.com/joomla/joomla-cms/issues/46111
